### PR TITLE
Hindi Translation for quick-start

### PR DESCRIPTION
**Pull request by akramkazmi71**
Hindi translation for docs/quick-start. issue #1

### CODE REVIEW

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
@akramkazmi71 Thank you for working on this, but looks like you have created a new file for this in place of editing the existing one.
You need to move this file to `docs/` folder, currently its in `docs/docs/`

File: docs/docs/quick-start.md [hunk 1]
```diff
@@ -1,66 +1,66 @@
 ---
-title: Quick Start
+शीर्षक: क्विक स्टार्ट
 ---
 
-This quick start is intended for intermediate to advanced developers. For a gentler intro to Gatsby, [head to our tutorial](/tutorial/)!
+यह क्विक स्टार्ट मध्यवर्ती से उन्नत डेवलपर्स के लिये है| गैट्सबी के एक विनम्र परिचय के लिए, [हमारे ट्यूटोरियल देखें](/tutorial/)!
```
Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `मध्यवर्ती` => `इंटरमीडिएट`

Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `उन्नत` => `एडवांस्ड`

Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `है|` => `है।`

File: docs/docs/quick-start.md [hunk 2]
```diff
@@ -1,66 +1,66 @@
...
 
-## Use the Gatsby CLI
+## गैट्सबी सीएलआई का उपयोग
```
Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `गैट्सबी` => `Gatsby`

We are not translating `Gatsby` word, Can you find and replace this in whole file

Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `सीएलआई का उपयोग` => `सीएलआई का उपयोग करें`

File: docs/docs/quick-start.md [hunk 3]
```diff
@@ -1,66 +1,66 @@
...
 
 <EggheadEmbed
   lessonLink="https://egghead.io/lessons/gatsby-quick-start-with-gatsby-create-develop-and-build-gatsby-sites-from-the-command-line"
-  lessonTitle="Quick Start with Gatsby: Create, Develop, and Build Gatsby Sites From the Command Line"
+  lessonTitle="गैट्सबी के साथ तुरंत शुरुआत करें: कमांड लाइन से गैट्सबी साइट बनाएँ, विकसित और निर्माण करें"
```
Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `विकसित` => `विकसित करें`

File: docs/docs/quick-start.md [hunk 4]
```diff
@@ -1,66 +1,66 @@
...
 />
 
-**Note**: this video uses `npx`, which is a tool to execute an npm package without first installing it. Running the command `npx gatsby new` is the same as running `gatsby new` after installing the gatsby-cli on your computer.
+**नोट**: इस वीडियो में `npx` का उपयोग किया गया है, जो पहले स्थापित किए बिना npm पैकेज को निष्पादित करने के लिए एक उपकरण है। अपने कंप्यूटर पर गैट्सबी-सीएलआई को स्थापित करने के बाद कमांड `npx gatsby new` को चलाना `gatsby new` चलाने के समान है।
```
Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `स्थापित` => `इनस्टॉल`

Change this everywhere, (find & replace)

Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `निष्पादित` => `एग्ज़ीक्यूट`

Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `उपकरण` => `टूल`

Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `गैट्सबी` => `Gatsby`

File: docs/docs/quick-start.md [hunk 5]
```diff
@@ -1,66 +1,66 @@
...
 
-### Install the Gatsby CLI.
+### गैट्सबी सीएलआई स्थापित करें।
```
Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `गैट्सबी` => `Gatsby`

Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `स्थापित` => `इनस्टॉल`

File: docs/docs/quick-start.md [hunk 6]
```diff
@@ -1,66 +1,66 @@
...
 
-### Change directories into site folder.
+### डायरेक्ट्रीज को साइट फ़ोल्डर में बदलें।
 
 ```shell
 cd gatsby-site
 ```
 
-### Start development server.
+### डेवलपमेंट सर्वर शुरू करें|
```
Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `करें|` => `करें।`

File: docs/docs/quick-start.md [hunk 7]
```diff
@@ -1,66 +1,66 @@
...
 
 ```shell
 gatsby develop
 ```
 
-Gatsby will start a hot-reloading development environment accessible by default at `localhost:8000`.
+गैट्सबी डिफ़ॉल्ट रूप से हॉट-रीलोडिंग डेवलपमेंट वातावरण शुरू करेगा `localhost:8000`.
```
Comment (arshadkazmi42) [inline docs/docs/quick-start.md]:
```
गैट्सबी डिफ़ॉल्ट रूप से हॉट-रीलोडिंग डेवलपमेंट वातावरण शुरू करेगा `localhost:8000`.
```
=> 
```
Gatsby एक हॉट-रीलोडिंग डेवलपमेंट एनवायरनमेंट सुरु करेगा जो की डिफ़ॉल्ट रूप से `localhost:8000` पर एक्सेसिबल रहेगाें।
```

File: docs/docs/quick-start.md [hunk 8]
```diff
@@ -1,66 +1,66 @@
...
 
-Try editing the JavaScript pages in `src/pages`. Saved changes will live reload in the browser.
+`Src / pages` में जावास्क्रिप्ट पेज को एडिट करने का प्रयास करें। बचाया गए परिवर्तन ब्राउज़र में पुनः लोड होंगे।
```
Comment (arshadkazmi42) [inline docs/docs/quick-start.md]:
```
`Src / pages`
```
=>
```
`src/pages`
```

Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `बचाया गए परिवर्तन ब्राउज़र में पुनः लोड होंगे` => `सेव किये गए बदलाव ब्राउज़र में लाइव पुनः लोड होंगे`

File: docs/docs/quick-start.md [hunk 9]
```diff
@@ -1,66 +1,66 @@
...
 
-### Create a production build.
+### एक प्रोडक्शन बिल्ड बनाएँ।
 
 ```shell
 gatsby build
 ```
 
-Gatsby will perform an optimized production build for your site, generating static HTML and per-route JavaScript code bundles.
+गैट्सबी स्थैतिक HTML और प्रति मार्ग जावास्क्रिप्ट कोड बंडलों का निर्माण करते हुए, आपकी साइट के लिए एक अनुकूलित उत्पादन निर्माण करेगा।
```
Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `गैट्सबी` => `Gatsby`

Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `स्थैतिक` => `इसटैटिक`

Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `अनुकूलित उत्पादन निर्माण करेगा` => `ऑप्टीमाइज़्ड प्रोडक्शन बिल्ड का निर्माण करेगा`

File: docs/docs/quick-start.md [hunk 10]
```diff
@@ -1,66 +1,66 @@
...
 
-### Serve the production build locally.
+### उत्पादन का निर्माण स्थानीय स्तर पर करें।
```
Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `उत्पादन का निर्माण स्थानीय स्तर पर करें।` => `प्रोडक्शन बिल्ड को लोकल पर सर्व करें।`

File: docs/docs/quick-start.md [hunk 11]
```diff
@@ -1,66 +1,66 @@
...
 
 ```shell
 gatsby serve
 ```
 
-Gatsby starts a local HTML server for testing your built site. Remember to build your site using `gatsby build` before using this command.
+गैट्सबी आपकी निर्मित साइट के परीक्षण के लिए एक स्थानीय HTML सर्वर शुरू करता है। इस कमांड का उपयोग करने से पहले `gatsby build` का उपयोग करके अपनी साइट बनाना याद रखें।
```
Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `गैट्सबी` => `Gatsby`

Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `निर्मित` => `बनी हुई`

Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `स्थानीय` => `लोकल`

File: docs/docs/quick-start.md [hunk 12]
```diff
@@ -1,66 +1,66 @@
...
 
-### Access documentation for CLI commands
+### सीएलआई कमांड के लिए डॉक्यूमेंटेशन
```
Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `डॉक्यूमेंटेशन` => `एक्सेस डॉक्यूमेंटेशन`

File: docs/docs/quick-start.md [hunk 13]
```diff
@@ -1,66 +1,66 @@
...
 
-To see detailed documentation for the CLI commands, run `gatsby --help` in the terminal.
+सीएलआई कमांड के लिए विस्तृत दस्तावेज देखने के लिए, टर्मिनल में `gatsby --help` चलाएं।
```
Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `कमांड के लिए विस्तृत दस्तावेज` => `कमांड के विस्तृत डॉक्यूमेंटेशन`

File: docs/docs/quick-start.md [hunk 14]
```diff
@@ -1,66 +1,66 @@
...
 
-For specific commands, run `gatsby COMMAND_NAME --help` e.g. `gatsby new --help`.
+विशिष्ट आदेशों के लिए, `gatsby COMMAND_NAME --help` रन करें, उदाहरण: `gatsby new --help`|
```
Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `आदेशों` => `कमांड्स`

Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `रन करें` => `चलाएं`

Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `|` => `।`

File: docs/docs/quick-start.md [hunk 15]
```diff
@@ -1,66 +1,66 @@
...
 
-For more information on the Gatsby CLI, visit the [CLI reference](/docs/gatsby-cli/) section of the docs.
+गैट्सबी सीएलआई के बारे में अधिक जानकारी के लिए, [सीएलआई संदर्भ](/docs/gatsby-cli/) देखें|
```
Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `गैट्सबी` => `Gatsby`

Comment (arshadkazmi42) [inline docs/docs/quick-start.md]: `|` => `।`

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Great work.
Added few feedbacks.
It will be good to go, once those fixes are check in 👍

**Comment by akramkazmi71** [comment]
Updated the doc as specified.

File: docs/docs/quick-start.md (hunk 7 above)
Comment (arshadkazmi42) [reply docs/docs/quick-start.md]: `रहेगाें` => `रहेगा`

Comment (akramkazmi71) [reply docs/docs/quick-start.md]: resolved.

**Review by arshadkazmi42 — APPROVED** [review]
Great. Thank you 👍

**Comment by arshadkazmi42** [comment]
@akramkazmi71 You can grab your swag from above link. 🎉 
Thank you for taking time and working on this translation

**Comment by imrishabh18** [comment]
Hey, I am a beginner. Can i contribute to translate any doc ?

**Comment by monizb** [comment]
Hello @imrishabh18 ! Welcome to Gatsby. Yes, you can contribute to any doc which is not yet taken by anyone. Please head over to https://github.com/gatsbyjs/gatsby-hi/issues/1 and comment the name of the doc you want to translate and then one of us will assign them to you :)

**Pull request merged**